HULKs/hulk
# Head and body, body and head

## Introduced Changes

This PR fixes the swapped body and head IDs for 33 and the swapped body/head IDs returned by aliveness.

## ToDo / Known Issues

None.

## Ideas for Next Iterations (Not This PR)

None.

## How to Test

```
pepsi playernumber set 33:1
pepsi upload
```

Confirm player number assignment (e.g. using twix).
